app/mailers/bulk_upload_mailer.rb

```diff
@@ -27,9 +27,9 @@ def send_how_fix_upload_mail(bulk_upload:)
 
   def send_bulk_upload_complete_mail(user:, bulk_upload:)
     url = if bulk_upload.lettings?
-            lettings_logs_url
+            "#{lettings_logs_url}?years[]=&status[]=&user=all&organisation_select=all"
```

Comment:
not related to this ticket, ensures when bulk upload logs are successfully uploaded the email links the user to the logs page with the filter “completed” applied so that the completed logs actually appear

Reply:
Please could you build the params with the URL helper? It should be possible to do this without string interpolation.

Reply:
yep, was struggling to get the years/status section to show up using the helper but have figured it out

Reply:
It's very much working as far as I can see - I think @SamSeed-Softwire is referring to the check box not being ticked rather than the filter actually being applied, but I'm not convinced that's something we want to change, and if so it's definitely not in the scope of this PR.

Lmk if I've misunderstood that sam and we can take a look together why it's not working as intended :)